src/code/audio/code_8019AF00.c

```diff
@@ -1878,6 +1879,8 @@ const u8 sIsOcarinaSongReserved[OCARINA_SONG_MAX] = {
 };
 */
 
+s32 AudioOcarina_MemoryGameGenerateNotes(void);
```

Comment:
Is there a reason this prototype is here specifically?

Reply:
I wanted to have it under the local struct defines but taking a second look, I put it all the way below the data. It can be moved above the data